packages/elements/src/components/ino-select/ino-select.tsx

```diff
@@ -131,23 +157,32 @@ export class Select implements ComponentInterface {
       'mdc-select--required': this.required,
     });
 
-    const hiddenInput = this.required ? (
-      <input
-        ref={(el) => (this.nativeInputElement = el)}
-        required={this.required}
-      ></input>
-    ) : (
-      ''
-    );
-
     return (
       <Host name={this.name}>
         <div class={classSelect}>
           <div class="mdc-select__anchor">
-            <div class="mdc-select__selected-text">
-              {this.value}
-              {hiddenInput}
-            </div>
+            <div class="mdc-select__selected-text">{this.value}</div>
+            <select
+              style={{ left: '50px', top: '50px' }}
```

Comment:
Why do we need these inline styles?

Reply:
Oh, thx I think this was just for exploring purposes. :)